services/api/service.go

```diff
@@ -145,8 +145,8 @@ type RelayAPI struct {
 	capellaEpoch   uint64
 
 	proposerDutiesLock       sync.RWMutex
-	proposerDutiesResponse   []boostTypes.BuilderGetValidatorsResponseEntry
-	proposerDutiesMap        map[uint64]*boostTypes.RegisterValidatorRequestMessage
+	proposerDutiesResponse   *[]byte // raw http response
```

Comment:
pointer to slice looks a little off to me. why not just use the slice object?

Reply:
because we want to replace it by just replacing the pointer, that way it's quicker (i.e. shorter time for needing the lock), isn't it?

Reply:
for the record -- it does work as expected (https://boost-relay-goerli.flashbots.net/relay/v1/builder/validators)